cmd/edit/cluster/cmd.go

```diff
@@ -189,9 +197,26 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
+	disableWorkloadMonitoring := cluster.DisableUserWorkloadMonitoring()
+
+	if cmd.Flags().Changed("disable-workload-monitoring") {
+		disableWorkloadMonitoring = args.disableWorkloadMonitoring
+	}
+	if interactive.Enabled() {
+		disableWorkloadMonitoring, err = interactive.GetBool(interactive.Input{
+			Question: "Disable Workload monitoring",
+			Help:     cmd.Flags().Lookup("disable-workload-monitoring").Usage,
+			Default:  disableWorkloadMonitoring,
+		})
+		if err != nil {
+			reporter.Errorf("Expected a valid disable-workload-monitoring value: %v", err)
+			os.Exit(1)
+		}
+	}
 	clusterConfig := ocm.Spec{
-		Expiration: expiration,
-		Private:    private,
+		Expiration:                expiration,
+		Private:                   private,
+		DisableWorkloadMonitoring: &disableWorkloadMonitoring,
```

Comment:
Since `disableWorkloadMonitoring` is always defined (`true` or `false`), would it make sense to split this so that it only gets set if changed, like we do for `--private`
```go
	var disableWorkloadMonitoring *bool
	var disableWorkloadMonitoringValue bool

	if cmd.Flags().Changed("disable-workload-monitoring") {
		disableWorkloadMonitoringValue = args.disableWorkloadMonitoring
		disableWorkloadMonitoring = &disableWorkloadMonitoringValue
	} else if interactive.Enabled() {
		disableWorkloadMonitoringValue = cluster.DisableUserWorkloadMonitoring()
	}

	if interactive.Enabled() {
		disableWorkloadMonitoring, err = interactive.GetBool(...)
	}

	clusterConfig := ocm.Spec{
		...
		DisableWorkloadMonitoring: disableWorkloadMonitoring,
	}
```